build.sh

```diff
@@ -18,7 +18,7 @@ ARGS=$*
 # script, and that this script resides in the repo dir!
 REPODIR=$(cd $(dirname $0); pwd)
 
-VALIDARGS="clean cppraft pyraft -v -g --allgpuarch --nvtx --show_depr_warn -h --buildgtest"
+VALIDARGS="clean cppraft pyraft -v -g --allgpuarch --nvtx --show_depr_warn -h --buildgtest --buildfaiss"
```

Comment:
I think @cjnolet is actually reviewing the code itself, so just asking about this: why default to building faiss statically as opposed to using the conda package? Given that cuML already has the faiss conda package as a dependency, I think we should do the same here

Reply:
I'm fine with removing it as well. I added it for consistency since cuml has that option and we also do it for gtest

Reply:
Totally my mistake and for some reason thought it was defaulting to building static faiss by default, so my comment can be disregarded :)